Gopkg.toml

```diff
@@ -58,7 +58,7 @@
 
 [[constraint]]
   name = "github.com/tendermint/go-amino"
-  version = "v0.14.0"
```

Comment:
I find it kind of weird that this now break dependencies for any downstream application who was still using `v0.14.0`. perhaps we should set it to allow `v0.14.0` and `v0.14.1`, and then the downstream rasp-pi app would just select `v0.14.1`?

Reply:
Doesn't block merge, but is a general note about dependencies

Reply:
Yeah, I think we should switch to `^X.Y.Z` or `~X.Y.Z` for most of the dependencies.